measurement-plane/tcp-tls-throughput/client.go

```diff
@@ -0,0 +1,64 @@
+package tcpTlsThroughput
+
+import "sync"
+import "os"
+import "strconv"
+import "fmt"
+import "crypto/tls"
+import "github.com/protocollabs/mapago/control-plane/ctrl/shared"
+
+func NewTcpTlsMsmtClient(config shared.ConfigurationObj, msmtStartRep *shared.DataObj, wg *sync.WaitGroup, closeConnCh <-chan string, callSize int) {
+	lAddr := config.Listen_addr
+	serverPorts := shared.ConvStrToIntSlice(msmtStartRep.Measurement.Configuration.UsedPorts)
+
+	for _, port := range serverPorts {
+		listen := lAddr + ":" + strconv.Itoa(port)
+		wg.Add(1)
+		go tcpTlsClientWorker(listen, wg, closeConnCh, callSize)
+	}
+}
+
+func tcpTlsClientWorker(addr string, wg *sync.WaitGroup, closeConnCh <-chan string, callSize int) {
+	buf := make([]byte, callSize, callSize)
+	certPath := "/src/github.com/protocollabs/mapago/measurement-plane/tcp-tls-throughput/certs"
```

Comment:
no, this is not clean. The cert should be installed with the install routine somewhere else. But ok for now